routes/api.js

```diff
@@ -8,9 +8,9 @@
 
 'use strict';
 
-var expect = require('chai').expect;
-var MongoClient = require('mongodb').MongoClient;
-var ObjectId = require('mongodb').ObjectId;
+const expect = require('chai').expect;
+const MongoClient = require('mongodb').MongoClient;
+const ObjectId = require('mongodb').ObjectId;
 const MONGODB_CONNECTION_STRING = process.env.DB;
 //Example connection: MongoClient.connect(MONGODB_CONNECTION_STRING, function(err, db) {});
```

Comment:
I believe these lines can be removed from the boilerplate as users can write their own code to satisfy the route requirements.

Reply:
Agreed.